src/main/asciidoc/index.adoc

```diff
@@ -308,3 +308,32 @@ WARNING: The handler WILL NOT BE CALLED if sent publish packet with QoS=0.
 ----
 {@link examples.VertxMqttClientExamples#example9}
 ----
+
+=== Use proxy protocol
+
+[source,$lang]
+----
+{@link examples.VertxMqttClientExamples#example12}
```

Comment:
@sdvdxl this is the problem as pointed by @vietj

Reply:
fixed